### PR TITLE
[IMP] hr: 'orgChart' review and optimize SCSS

### PR DESCRIPTION
**Pull request by stefanorigano**
requires:

- https://github.com/odoo/odoo/pull/87687 -> merged
- https://github.com/odoo/enterprise/pull/25781 -> merged

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by stefanorigano** [comment]
Hello @pparidans , ready-ish for tech review.
It requires the `searchPanel` ones that are currently red... I pinged you there, thanks :)

**Comment by stefanorigano** [comment]
@pparidans ready for tech. review 👍

File: addons/web/static/src/legacy/scss/form_view.scss [hunk 1]
```diff
@@ -637,7 +637,7 @@ $o-form-label-margin-right: 0px;
         margin: 1.2em ($grid-gutter-width * .5) .8em 0;
 
         &:not(:empty) {
-            box-shadow: 0 1px 0 rgba($body-color, .05);
+            box-shadow: 0 1px 0 $o-form-separator-color;
```
Comment (pparidans) [inline addons/web/static/src/legacy/scss/form_view.scss:640]: I guess this should be also in non-legacy form view :wink:

File: addons/hr_org_chart/views/hr_views.xml [hunk 2]
```diff
@@ -6,9 +6,17 @@
...
         <field name="arch" type="xml">
             <div id="o_work_employee_main" position="after">
-                <div id="o_employee_right">
-                    <h4 class="o_org_chart_title mb16 mt0">Organization Chart</h4>
-                    <field name="child_ids" widget="hr_org_chart" readonly="1"/>
+                <div id="o_employee_right" class="col-lg-4 px-0 ps-lg-5 pe-lg-0">
+                    <table class="o_group o_inner_group">
+                        <tbody>
+                            <tr>
+                                <td class="pe-lg-0">
```
Comment (pparidans) [inline addons/hr_org_chart/views/hr_views.xml:13]: I'm confused: why adding `<group>` like HTML? And specially for a 1line/1cell table? :thinking:

File: addons/web/static/src/legacy/scss/form_view.scss (hunk 1 above)
Comment (stefanorigano) [reply addons/web/static/src/legacy/scss/form_view.scss:640]: yep, thanks :)

File: addons/hr_org_chart/views/hr_views.xml (hunk 2 above)
Comment (stefanorigano) [reply addons/hr_org_chart/views/hr_views.xml:13]: To ensure margin consistency with other formView elements.
I don't like it neither but it's the cleanest approach until we refactor the formView design :/
image

File: addons/hr_org_chart/views/hr_views.xml [hunk 3]
```diff
@@ -6,9 +6,17 @@
...
+                                    <div class="o_horizontal_separator o_org_chart_title">Organization Chart</div>
+                                    <field name="child_ids" class="position-relative" widget="hr_org_chart" readonly="1"/>
+                                </td>
+                            </tr>
+                        </tbody>
+                    </table>
```
Comment (pparidans) [inline addons/hr_org_chart/views/hr_views.xml:19]: What do you think about:
```suggestion
                    
                    <group>
                        <div class="o_horizontal_separator o_org_chart_title" colspan="2">Organization Chart</div>
                        <field name="child_ids" class="position-relative" widget="hr_org_chart" readonly="1" nolabel="1"/>
                    </group>
```

Comment (stefanorigano) [reply addons/hr_org_chart/views/hr_views.xml]: Confirmed, much better, thanks.
Implemented

Cc. @adr-odoo

**Comment by stefanorigano** [comment]
ok, @adr-odoo the mixin has been removed, thanks again 👍 

about this https://github.com/odoo/odoo/pull/93049#discussion_r938445857,  I now remember that generate un undesired padding-right.
Not a big deal though, ok for me. 
image

cc. @pparidans

**Review by pparidans — APPROVED** [review]
As discussed, LGTM :+1: 
@robodoo r+

**Comment by adr-odoo** [comment]
@robodoo rebase-ff

**Comment by xmo-odoo** [comment]
@robodoo retry

Runbot apparently kinda just died over the weekend, so timeout is likely not legit.

**Pull request closed without merging**